### PR TITLE
tutorial is now more focused

### PR DESCRIPTION
**Pull request by timeyyy**
Hopefully its more easy and straight forward to follow.
Also works on all platforms and python2/3

### CODE REVIEW

File: tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+applications. You will need a freezer module installed.
+
+This guide will focus on telling you what to do and to get your feet wet,
+jump into the source code of the examples for the details.
+
+## Choosing a Freezer
+
+Esky currently supports
+ - cx_freeze
+ - py2exe,
```
Comment (rfk) [inline tutorial/README.md]: spurious comma?

File: tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+We suggest that you use **cx_freeze** because it is/supports:
+ - python2
+ - python3
+ - maintained 
+ - linux
+ - mac 
+ - windows
+
+Esky is currently not working with py2exe, patches are welcome!,
```
Comment (rfk) [inline tutorial/README.md]: comma

File: tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+Bbfreeze is unmaintained!
```
Comment (rfk) [inline tutorial/README.md]: Perhaps we should just tip it out then, there's no good reason to keep maintaining our support here.  Can you please add more words here like "...and support will likely be removed in a future release of esky"?

File: tutorial/stage2/setup.py [hunk 4]
```diff
@@ -1,31 +1,40 @@
...
-    )
+# Using py2app
+#setup(
+#    name = "example-app",
+#    version = "0.2",
+#    scripts = ["example.py"],
+#    options = {"bdist_esky": {
+#                "freezer_module":"py2app"
+#             }}
+#    )
```
Comment (rfk) [inline tutorial/stage2/setup.py]: I wonder if we should break these out into separate subdirectories, one per freezer module?

Comment (rfk) [inline tutorial/stage2/setup.py]: (as a follow-up bug, not necessarily in this PR)

**Comment by rfk** [comment]
Thanks @timeyyy, this is certainly an improvement.  I had a couple of tiny nits but otherwise this looks good.

File: tutorial/stage2/setup.py (hunk 4 above)
Comment (timeyyy) [inline tutorial/stage2/setup.py]: sure thing.

**Comment by rfk** [comment]
@timeyyy looks like this has a merge conflict, can you please rebase and I'll merge?

**Pull request merged**